lib/beaker/hypervisor/docker.rb

```diff
@@ -102,7 +113,7 @@ def provision
         fix_ssh(container) if @options[:provision] == false
 
         @logger.debug("Starting container #{container.id}")
-        container.start({"PublishAllPorts" => true, "Privileged" => true})
```

Comment:
If any options are passed with `container.start` then those are the only options that are used. So originally any options provided by `container_opts` were not being used at all and therefore consolidating all the options in `container_opts`